requirements.txt

```diff
@@ -1,2 +1,2 @@
-strawberryfields>=0.11.2
-pennylane>=0.7
+git+https://github.com/XanaduAI/strawberryfields.git
+git+https://github.com/XanaduAI/pennylane.git@probs_cv_cutoff
```

Comment:
temporary till the content of `probs_cv_cutoff` is incorporated in PL (this allows the `cutoff` value to be used when returning probabilities)